src/utils/aquarius.ts

```diff
@@ -25,5 +28,63 @@ export async function getAsset(
     } else {
       LoggerInstance.error(error.message)
     }
+    throw error
+  }
+}
+
+export async function getAssets(
+  metadataCacheUri: string,
+  dids: string[]
+): Promise<{ [key: string]: Asset }> {
```

Comment:
nitpick: we could rename `key` to `did` to indicate to the user how these are used